api-contracts/openapi/components/schemas/v1/task.yaml

```diff
@@ -48,6 +48,12 @@ V1TaskSummary:
       type: string
       format: date-time
       description: The timestamp the task run started.
+    stepId:
```

Comment:
did we decide we want to start naming things consistently as task? or is step the defn and not the run in this instance?

Reply:
yeahh this is the actual step - we need it because the olap runs rows don't get created in time, so we need the steps themselves to sketch out the dag / minimap ui, then we hydrate those once more data on the runs comes back once the runs happen